docs-ref-conceptual/includes/cli-install-linux-script.md

```diff
@@ -15,7 +15,7 @@ It's strongly recommend to install the CLI with a package manager. A package man
 
 The CLI requires the following software:
 
-* [Python 3.6.x, 3.7.x or 3.8.x](https://www.python.org/downloads/).
+* [Python 3.6.x, 3.7.x or 3.8.x](https://www.python.org/downloads/). This install script is not written to work with Python 3.12.
```

Comment:
Python 3.6 and 3.7 support has been dropped long ago. We also added support for Python 3.9 ~ 3.11.

Reply:
This script is not compatible with Python 3.11 https://github.com/Azure/azure-cli/issues/27775